src/Drawables/DrawablePath.vala

```diff
@@ -37,11 +37,14 @@ public class Akira.Drawables.DrawablePath : Drawable {
             return;
         }
 
+        Geometry.Point[] translated_points = recalculate_points ();
```

Comment:
it's probably better to do this on construction, it doesn't seem like there is a reason to do it every time the path is created. If you want to keep the original points, you can just keep both the original and the recalculated points.